src/jvm.jl

```diff
@@ -16,17 +16,47 @@ const JNI_ENOMEM       = convert(Cint, -4)              #/* not enough memory */
 const JNI_EEXIST       = convert(Cint, -5)              #/* VM already created */
 const JNI_EINVAL       = convert(Cint, -6)              #/* invalid arguments */
 
+function javahome_winreg()
+    processor_architecture = ENV["PROCESSOR_ARCHITECTURE"]
+    keypath = utf16("")
+    javahome = ""
+    cv = "CurrentVersion"
+    jh = "JavaHome"
+    global const msvclib = "msvcr100.dll"
+    
+    if processor_architecture == "AMD64" 
+        if WORD_SIZE == 64
+            keypath = utf16("SOFTWARE\\JavaSoft\\Java\ Development\ Kit")
+        elseif WORD_SIZE == 32
+            keypath = utf16("SOFTWARE\\WoW6432Node\\JavaSoft\\Java\ Development\ Kit")
+        end
+    elseif processor_architecture == "x86"
+        if WORD_SIZE == 32
+            keypath = utf16("SOFTWARE\\JavaSoft\\Java\ Development\ Kit")
+        end
+    end
+
+    value = querykey(WinReg.HKEY_LOCAL_MACHINE, keypath, cv)
+
+    keypath *= utf16("\\")*utf16(value)
+
+    return querykey(WinReg.HKEY_LOCAL_MACHINE, keypath, jh)
+end
 
 @unix_only global const libname = "libjvm"
 @windows_only global const libname = "jvm"
+
 function findjvm()
     javahomes = Any[]
     libpaths = Any[]
-    
+    msvcpaths = Any[]
+      
+    @windows_only push!(javahomes, javahome_winreg())
+
     if haskey(ENV,"JAVA_HOME")
         push!(javahomes,ENV["JAVA_HOME"])
     end
-    if isfile("/usr/libexec/java_home")
+    if isexecutable("/usr/libexec/java_home")
```

Comment:
Had my local environment working from the latest tagged release of JavaCall (0.3.4), not master.  Will fix.